### ISSUE TITLE
config.json file unconfigured upon installation

### ISSUE THREAD

**Issue by gvlx** [bkw777/mainline#204]
Hello,

I don't know if this is by design, but my `~/.config/mainline/config.json` was missing the `"ppa_uri"` definition after installation.

As I saw no reference, issue or wiki entry about this, I was stuck as which value should be written.

So maybe as an documentation enhancement, it should be specified the bare `config.json` file should look like this:

```
{
  "ppa_uri" : "http://kernel.ubuntu.com/~kernel-ppa/mainline/",
  "all_proxy" : null,
  "notify_major" : "false",
  "notify_minor" : "false",
  "hide_unstable" : "false",
  "previous_majors" : "2",
  "notify_interval_unit" : "0",
  "notify_interval_value" : "0",
  "connect_timeout_seconds" : "30",
  "concurrent_downloads" : "0",
  "window_width" : "968",
  "window_height" : "879",
  "window_x" : "0",
  "window_y" : "0"
}
``` 

Thank you for maintaining this package, it really makes all the difference.

**Comment by gvlx** [comment]
Note: it should also be indicated that the first run (updating all the indexes) can take a _lot_ of time. As the progress bar is kept at "0%" most of the time, unaware first-time users might mistake this as an error and stop the program before it finishes.

**Comment by bkw777** [comment]
Looks like I broke something in one of the last few updates. It was definitely writing itself from a built-in default value within the last few days, but you're right, today it is not.

As for the progress up until recently it did behave like you say, but should be showing a counting progress as it goes since the last few updates.

I'll fix it.

**Comment by bkw777** [comment]
fixed in c69ecf7
